telemetry/definitions/commonDefinitions.json

```diff
@@ -367,10 +367,25 @@
             "description": "A general field for logging metadata associated to Amazon Q transform metrics."
         },
         {
-            "name": "codeTransformNumberOfProjects",
+            "name": "codeTransformNumberOfSelectedProjects",
             "type": "int",
             "description": "The number of projects selected for transformation"
         },
+        {
+            "name": "codeTransformNumberOfTransformedFiles",
+            "type": "int",
+            "description": "The number of files that were transformed during a transformation job"
+        },
+        {
+            "name": "codeTransformNumberOfTransformedFilesAccepted",
```

Comment:
https://github.com/aws/aws-toolkit-common/blob/21a260645545cb008992e47b768070353beaf18e/telemetry/definitions/commonDefinitions.json#L42-L51

are these the same fields? if your team is changing types, please make sure you clean the ones you dont reuse.

Reply:
Is there a reason to capture these on the IDE side? Elastic Gumby backend already capture these metrics.

Reply:
This is exclusive to .NET Transform. We are essentially trying to track "out of the number of changes that the customer sees, how many do they accept?". This information can only be tracked on the IDE where the user interacts.

If changing the metrics names would help, do you have suggestions?

Reply:
Got it. I'm wondering if your team can reuse codeTransform_viewArtifact, which indicates users have interacted with the version control of the upgraded code. This can be added as a new (optional) field in it.

Reply:
are you planning to remove `amazonqNumberOfFilesGenerated` & `amazonqNumberOfFilesAccepted` repeating metrics?

Reply:
Those fields are being utilized by metric types that we have not defined, so I haven't removed them in case that would be a breaking change.